docs_outdoornav_user_manual/getting_started/terminal_interface.mdx

```diff
@@ -0,0 +1,122 @@
+the system or who want more precise control for managing the Navigation
+component.
+
+### Connecting to the OutdoorNav Computer {#connecting-to-outdoornav-computer}
+
+To connect to your UGV, consult its corresponding user manual.
+
+If you are using a Clearpath Robotics UGV with an OutdoorNav computer,
+first `ssh` to the UGV using the details provided in the UGV user
+manual. If you have a wired connection to the Clearpath Robotics UGV,
+use the following command. If using wifi, you can replace the IP address
+with the wifi-assigned IP address or the hostname of the UGV.
+
+``` bash
+ssh administrator@192.168.131.1
+```
+
+Then, connect to the OutdoorNav Computer:
+
+``` bash
+ssh administrator@192.168.131.5
+```
+
+### Starting the Navigation Software {#starting-outdoornav}
+
+Begin by connecting to the OutdoorNav Computer as outlined above.
+
+On UGV startup, all the sensors, the user interface, as well as the
+Navigation software are set to start automatically through a Docker
+container. You can check the Docker container's status by running
+`docker compose ps` and checking for:
+
+-   onav-web (Docker image containing the web interface)
+-   onav-web-ros (Docker image containing the ROS web bridge nodes)
+-   onav-sensors (Docker image containing that launches the ROS sensor
+    drivers)
+-   onav-autonomy (Docker image containing the ROS nodes related to the
+    autonomy)
+
+If the Docker containers are not running, they can be started with:
+
+``` bash
+cd ~/cpr_outdoornav_launch/    # directory with docker-compose.yaml
+docker compose --profile outdoornav up -d
+```
+
+### Stopping/Restarting all of OutdoorNav
+
+To use the UGV without the OutdoorNav software, use these commands to
+stop the software and prevent it from automatic startup:
+
+``` bash
+cd ~/cpr_outdoornav_launch/    # directory with docker-compose.yaml
+docker compose --profile outdoornav stop
+```
+
+If the OutdoorNav software is currently running or has been stopped, it
+can be restarted by running:
+
+``` bash
+cd ~/cpr_outdoornav_launch/    # directory with docker-compose.yaml
+docker compose --profile outdoornav start
+```
+
+### Stopping/Restarting the Autonomy
+
+To use the UGV without the autonomy core of OutdoorNav, use these
+commands to stop the nodes and prevent them from automatic startup:
+
+``` bash
+cd ~/cpr_outdoornav_launch/    # directory with docker-compose.yaml
+docker compose --profile autonomy stop
+```
+
+The autonomy core can be restarted by running:
+
+``` bash
+cd ~/cpr_outdoornav_launch/    # directory with docker-compose.yaml
+docker compose --profile outdoornav start
+```
+
+### Stopping/Restarting the Sensors
+
+To use the UGV without the sensors, use these commands to disable the
+nodes and prevent them from automatic startup:
+
+``` bash
+cd ~/cpr_outdoornav_launch/    # directory with docker-compose.yaml
+docker compose stop
+```
+
+### Accessing the Shell in Docker Container

```

Comment:
Probably want an INFO or WARNING section here to note that customers can break things easily and should be careful if reviewing things inside the containers.

Reply:
I would completely remove this section and completely prevent the customer that don't know they can enter the container from doing so. Any objections?

Reply:
Ok with me